static/app/utils/replays/frame.tsx

```diff
@@ -183,129 +277,3 @@ function stringifyNodeAttributes(node: SlowClickFrame['data']['node']) {
       : ''
   }`;
 }
-
-export function getDescription(frame: ReplayFrame): ReactNode {
-  if ('category' in frame) {
-    switch (frame.category) {
-      case 'replay.init':
-        return stripOrigin(frame.message ?? '');
-      case 'navigation':
-        const navFrame = frame as NavFrame;
-        return stripOrigin(navFrame.data.to);
-      case 'issue':
-      case 'ui.slowClickDetected': {
```

Comment:
This was a bug before.

The issue case was falling through to slow-click case, therefore the issue description was very wrong in a few places :(